bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -1038,6 +1063,7 @@ class PresentationUploader extends Component {
 
           <div className={styles.modalHint}>
             {`${intl.formatMessage(intlMessages.message)}`}
+            {this.renderExtraHint()}
```

Comment:
Could this be optional, controlled in settings.yml ? Something like

`displayHintAboutPresentationConstraints: false`